docs/wiki/archive/Magnetometer.mdx

```diff
@@ -17,56 +20,65 @@ Mag information is essential for position hold, which we intend to support.
+:::
 
-![Mag axes and orientation](../images/MagOrientationDiagram.png)
-Fig 1 - Mag axes and orientation
+<figure className="align-center">
+  <img src={MagOrientation} alt="Figure of magnetometer axes and orientation" className="no-effect " />
+  <figcaption>
+    <b>Fig. 1</b> - Magnetometer axes and orientation in relation to the drone as expected by Betaflight
+  </figcaption>
+  <br />
+</figure>
 
-If the magnetometer cannot be physically positioned so that it's axes are aligned as per the diagram above, the `Mag alignment` setting in Configurator (`align_mag` in CLI) may be used to correct for unusual Mag orientations. For example, some GPS units mount the mag unit upside down.
+If the magnetometer cannot be physically positioned so that its axes are aligned as per the diagram above, the `Mag alignment` setting in Configurator (`align_mag` in the [CLI](../configurator/cli-tab)) may be used to correct for other mag orientations. For example, some GPS units mount the mag unit upside down and some mag models use an unusual axis orientation.
 
 Correct orientation for a given axis can be confirmed by checking the Magnetometer data in the Sensors tab.
 
-## About the earth's magnetic field vector
+## About the Earth's Magnetic Vector Field
 
-The magnetic field of the earth doesn't simply point North and South, and is not parallel to the ground.
+[Earth's magnetic field](https://en.wikipedia.org/wiki/Earth%27s_magnetic_field)[^1] is not a uniform vector field. It is not simply oriented to the geographic North or South, and is not parallel to the ground in most places around the world. In other words: The field direction and field strength both vary from place to place.
 
-It is a vector that usually points a few degrees away from true North, and points either down into the ground in the Northern Hemisphere, or up out of the ground in the Southern Hemisphere. There are considerable angular deviations, depending upon where you are on the surface of the Earth.
+Earth's magnetic field usually gets represented in a **NED** (north-east-down) frame of reference. This means that the X axis points North, the Y axis points East, and the Z axis points Down. This is not at all relevant for Betaflight's mag orientation on the quad, but it's useful to know for interpreting the magnetic field at your location and for navigating around the earth's surface.
 
-The sideways deviation of the Earth's magnetic field from true north is called the `magnetic declination`. It is measured in degrees, with positive values meaning that the magnetic North is to the East of true North.
+If you measure the magnetic field at a particular point on earth, you will measure a vector, which usually points a few degrees away from the geographic North, and points either down into the ground in the Northern Hemisphere, or up out of the ground in the Southern Hemisphere. There are considerable angular deviations, depending upon where you are on the surface of the Earth.
 
```

Comment:
Maybe "you are measuring a vector which usually points a few degrees away from geographic North, and either down..."